### DISCUSSION TITLE
No matching distribution found for torch==0.4.1

### DISCUSSION THREAD

**Discussion by davidtvs** [community/community#26753] [Programming Help]
<p>I’m trying to run my CI job (<a href="https://github.com/davidtvs/pytorch-lr-finder/blob/dev/lr_history/.github/workflows/ci_build.yml#L15" rel="nofollow noopener">link</a>) against an older version of PyTorch (0.4.1) but I get the following error message when running <code>pip install torch==0.4.1</code>:</p>
<pre><code class="lang-auto">ERROR: Could not find a version that satisfies the requirement torch==0.4.1 (from versions: 0.1.2, 0.1.2.post1, 0.1.2.post2, 1.4.0, 1.5.0)
ERROR: No matching distribution found for torch==0.4.1
</code></pre>
<p>Here’s the <a href="https://github.com/davidtvs/pytorch-lr-finder/runs/721864064?check_suite_focus=true" rel="nofollow noopener">full log from the CI</a></p>
<p>Seems like github actions only has really old (0.1.2) or the newest versions (1.4.0 and 1.5.0)…is this right? If so, anyway to work around it?</p>
<p>Thanks</p>

**Comment by davidtvs** [comment, marked as answer]
<p>If anyone faces this problem, the solution is changing the pip install command to:<br>
<code>pip install torch==0.4.1 -f https://download.pytorch.org/whl/torch_stable.html</code></p>

**Reply by kvmukul** [reply]
your the best, thanks a lot

**Comment by jyflop** [comment]
<p>It’s still not working. please help</p>

**Reply by hrishik18** [reply]
If not, you might want to consider downgrading the Python version as an alternative solution.

**Comment by aymuos15** [comment]
This seemed to work for me
